index.js

```diff
@@ -113,6 +113,11 @@ const jwtSecret = process.env.JWT_SECRET;
 const schoolName = process.env.SCHOOL_NAME;
 const schoolDomain = process.env.SCHOOL_DOMAIN;
 const statsfileName = "data/data.json";
+let config = null;
+try {
+	config = require("./data/config.json");
+} catch (_) {}
+
```

Comment:
There has to be a better way for this

Reply:
Only if the app is supposed to crash when config file not exists

Reply:
There you go:

```
const fs = require('fs');

let configPath = './config.json';
let config = null;
if (fs.existsSync(configPath)) {
	config = require(configPath);
}
console.log(config);
```

Reply:
It just crashes now :smile: